app/screens/about/index.tsx

```diff
@@ -0,0 +1,351 @@
+        const onError = () => {
+            Alert.alert(
+                intl.formatMessage({
+                    id: 'mobile.link.error.title',
+                    defaultMessage: 'Error',
+                }),
+                intl.formatMessage({
+                    id: 'mobile.link.error.text',
+                    defaultMessage: 'Unable to open the link.',
+                }),
+            );
+        };
+
+        tryOpenURL(url, onError);
+    };
+
+    const handleAboutTeam = preventDoubleTap(() => {
+        return openURL(Config.AboutTeamURL);
+    });
+
+    const handleAboutEnterprise = preventDoubleTap(() => {
+        return openURL(Config.AboutEnterpriseURL);
+    });
+
+    const handlePlatformNotice = preventDoubleTap(() => {
+        return openURL(Config.PlatformNoticeURL);
+    });
+
+    const handleMobileNotice = preventDoubleTap(() => {
+        return openURL(Config.MobileNoticeURL);
+    });
+
+    const handleTermsOfService = preventDoubleTap(() => {
+        return openURL(AboutLinks.TERMS_OF_SERVICE);
+    });
+
+    const handlePrivacyPolicy = preventDoubleTap(() => {
+        return openURL(AboutLinks.PRIVACY_POLICY);
+    });
+
+    return (
+        <SafeAreaView
+            edges={['left', 'right']}
+            style={style.container}
+            testID='about.screen'
+        >
+            <StatusBar theme={theme}/>
+            <ScrollView
+                style={style.scrollView}
+                contentContainerStyle={style.scrollViewContent}
+                testID='about.scroll_view'
+            >
+                <View style={style.logoContainer}>
+                    <CompassIcon
+                        name='mattermost'
+                        color={theme.centerChannelColor}
+                        size={120}
+                        testID='about.logo'
+                    />
+                </View>
+                <View style={style.infoContainer}>
+                    <View style={style.titleContainer}>
+                        <Text
+                            style={style.title}
+                            testID='about.site_name'
+                        >
+                            {`${config.value.SiteName} `}
+                        </Text>
+                        <Title
+                            config={config}
+                            license={license}
+                        />
+                    </View>
+                    <Subtitle config={config}/>
+                    <FormattedText
+                        id='mobile.about.appVersion'
+                        defaultMessage='App Version: {version} (Build {number})'
+                        style={style.info}
+                        values={{
+                            version: DeviceInfo.getVersion(),
+                            number: DeviceInfo.getBuildNumber(),
+                        }}
+                        testID='about.app_version'
+                    />
```

Comment:
There is a component called `AppVersion` that you could use here

Reply:
added